### DISCUSSION TITLE
As a developer of what kind of open source project can continue to use the co-pilot for free?

### DISCUSSION THREAD

**Discussion by XiaoYingYo** [community/community#19754] [Copilot Conversations]
As a developer of what kind of open source project can continue to use the co-pilot for free? 
Fork number of times?
Star number of times?

**Comment by thomscoder** [comment]
I do not think there is a full fledged list of projects.
It's probably a combination of factors: stars, forks, contributors, used by etc...

I'd say a size of the impact the Organization would receive if you were to be "slowed down" by a 10$/month fee

**Comment by D7EAD** [comment]
Hi!

Github Copilot is available for free, as of right now, to verified students and "popular open source projects." What Github defines as a popular open source project is, sadly, not expressly stated.

**Comment by icdance** [comment]
They haven't yet disclosed the criteria for [open source maintainers](https://github.com/pricing#i-work-on-open-source-projects-can-i-get-access-to-github-copilot-for-free):

> People who maintain popular open source projects receive a credit to have 12 months of GitHub Copilot access for free. A maintainer of a popular open source project is defined as someone who has write or admin access to one or more of the most popular open source projects on GitHub. Simply visit the GitHub Copilot subscription page to see if you are one of the open source maintainers that meet our criteria for a complimentary subscription. If you do, you should see that you can add GitHub Copilot for no charge. If you see a charge on the purchase page then this means that you do not qualify at this time. Once awarded, if you are still a maintainer of a popular open source project when your initial 12 months subscription expires then you will be able to renew your subscription for free.

So it will be up to them to define the meaning of `popular`, maybe top 100, top 500?  so far the [reply ](https://www.reddit.com/r/github/comments/vjqh4b/comment/idnlmdy/?utm_source=share&utm_medium=web2x&context=3)we got

> The Copilot and billing teams will be working on adding an option for organizations, hopefully available later this year. We will definitely let folks know when that's an option, but for now, it is only available on a per-user basis, I'm afraid. 

sigh

**Comment by garmeeh** [comment]
I think they are hand picking projects rather than using any criteria. If there was a set list I think it would have been published. 

I have a project with over 5,000 stars and 250k weekly downloads and I do not qualify as an open source maintainer. I was using Co-Pilot in beta round also, but now would have to pay 🤷‍♂️

**Comment by ZGGSONG** [comment]
> I think they are hand picking projects rather than using any criteria. If there was a set list I think it would have been published.
> 
> I have a project with over 5,000 stars and 250k weekly downloads and I do not qualify as an open source maintainer. I was using Co-Pilot in beta round also, but now would have to pay 🤷‍♂️

It's really frustrating.
